src/components/TileSelect/TileOption.js

```diff
@@ -18,15 +18,15 @@ const TileOption = ({ displayName, value, logo, onChange, isSelected }) => {
         text-align: center;
         padding: 0.5rem;
         color: var(--system-text-primary-light);
-        border: 1px solid #3a444b;
+        border: ${isSelected
+          ? '1.25px solid var(--attention-notification-info)'
+          : ' 1px solid #3a444b'};
```

Comment:
this outline is hard to see in dark mode. not a showstopper but may want to ask design for suggestion
![2022-08-26_14-17-21](https://user-images.githubusercontent.com/2952843/186992068-46730de2-37ef-4bbc-a044-b4f023c9c6b6.png)
s